pkg/pillar/persistcache/persistcache.go

```diff
@@ -0,0 +1,146 @@
+// Copyright (c) 2023 Zededa, Inc.
+// SPDX-License-Identifier: Apache-2.0
+
+package persistcache
+
+import (
+	"io/fs"
+	"os"
+	"path"
+	"path/filepath"
+	"strings"
+	"sync"
+
+	fileutils "github.com/lf-edge/eve/pkg/pillar/utils/file"
+)
+
+type persistCache struct {
+	sync.Mutex
+	cache map[string][]byte
+	root  string
+}
+
+const FileMask = 0755
+const LockFileName = "persistcache.lock"
```

Comment:
I thought you removed the lock file, but it is still here and in the test

Reply:
removed it all